README.md

```diff
@@ -41,39 +41,41 @@ This tool expects to run in an environment with the following attributes:
 
 Requesting a code review:
 
-    git appraise request
+    git review request
 
 Pushing code reviews to a remote:
 
-    git appraise push [<remote>]
+    git review push [<remote>]
 
 Pulling code reviews from a remote:
 
-    git appraise pull [<remote>]
+    git review pull [<remote>]
 
 Listing open code reviews:
 
-    git appraise list
+    git review
+    git review --list
 
 Showing the status of the current review, including comments:
 
-    git appraise show
+    git review show
 
 Showing the diff of a review:
 
-    git appraise show --diff [--diff-opts "<diff-options>"] [<review-hash>]
+    git review show -p/-u/--patch [--diff-opts "<diff-options>"] [<review-hash>]
```

Comment:
Yes, it would just take the hash of the review. That's another reason I was hesitant to add a "diff" subcommand, but I don't think it should be a deal breaker.